names.idc

```diff
@@ -423,20 +522,118 @@ static initNames(void)
+    MakeName(0x004CEC25, "ViewportManager::collectGarbage");
     MakeName(0x004CEC50, "clip_drawpixelinfo");
     MakeName(0x004CF142, "set_cursor_to_1");
     MakeName(0x004CF167, "set_cursor");
+    MakeName(0x004CF194, "widget::draw_tab");
+    MakeName(0x004CF284, "company_id_from_dropdown");
+    MakeName(0x004CF2B3, "populate_company_dropdown");
+    MakeName(0x004CF487, "widget::drawViewportCentreButton");
     MakeName(0x004CF5C5, "progressbar_begin");
     MakeName(0x004CF60B, "progressbar_end");
     MakeName(0x004CF621, "progressbar_increment");
+    MakeName(0x004CFF1E, "j_DirectDrawCreateEx");
+    MakeName(0x004D0FA0, "j_DirectInputCreateA");
+    MakeName(0x004D0FA6, "j_DSOUND_1");
+    MakeName(0x004D0FAC, "j_DSOUND_2");
     MakeName(0x004D1090, "LeadUp1");
     MakeName(0x004D10BC, "LeadUp2");
     MakeName(0x004D1178, "TrailUp0");
+    MakeName(0x004D1F97, "start");
     MakeName(0x004D2D3F, "__cropzeros");
     MakeName(0x004D2D8D, "__positive");
     MakeName(0x004D46EC, "LeadUp2_0");
     MakeName(0x004D47A8, "TrailUp0_0");
     MakeName(0x004D728C, "wsprintfA");
+    MakeName(0x00508F04, "_gameCommandFlags");
+    MakeName(0x00508F10, "multiplayer_flags");
+    MakeName(0x0050AED7, "config_music_playing");
+    MakeName(0x0050AF27, "music_playlist");
+    MakeName(0x0050AF40, "enabled_music");
+    MakeName(0x0050BF58, "colour");
+    MakeName(0x0050C3D0, "_interfaceSkinObjects");
+    MakeName(0x0050C3D4, "_soundObjects");
+    MakeName(0x0050C5D4, "_currencyObjects");
+    MakeName(0x0050C5D8, "_steamObjects");
+    MakeName(0x0050C658, "_rockObjects");
+    MakeName(0x0050C678, "_waterObjects");
+    MakeName(0x0050C67C, "_landObjects");
+    MakeName(0x0050C6FC, "_townNamesObjects");
+    MakeName(0x0050C700, "_cargoObjects");
+    MakeName(0x0050C780, "_wallObjects");
+    MakeName(0x0050C800, "_trackSignalObjects");
+    MakeName(0x0050C840, "_levelCrossingObjects");
+    MakeName(0x0050C850, "_streetLightObjects");
+    MakeName(0x0050C854, "_tunnelObjects");
+    MakeName(0x0050C894, "_bridgeObjects");
+    MakeName(0x0050C8B4, "_trackStationObjects");
+    MakeName(0x0050C8F4, "_trackExtraObjects");
+    MakeName(0x0050C914, "_trackObjects");
+    MakeName(0x0050C934, "_roadStationObjects");
+    MakeName(0x0050C974, "_roadExtraObjects");
+    MakeName(0x0050C984, "_roadObjects");
+    MakeName(0x0050C9A4, "_airportObjects");
+    MakeName(0x0050C9C4, "_dockObjects");
+    MakeName(0x0050C9E4, "_vehicleObjects");
+    MakeName(0x0050CD64, "_treeObjects");
+    MakeName(0x0050CE64, "_snowObjects");
+    MakeName(0x0050CE68, "_climateObjects");
+    MakeName(0x0050CE6C, "_hillShapesObjects");
+    MakeName(0x0050CE70, "_buildingObjects");
+    MakeName(0x0050D070, "_scaffoldingObjects");
+    MakeName(0x0050D074, "_industryObjects");
+    MakeName(0x0050D0B4, "_regionObjects");
+    MakeName(0x0050D0B8, "_competitorsObjects");
+    MakeName(0x0050D138, "_scenarioTextObjects");
```

Comment:
Not sure whether these names are already covered by one of the other files — I couldn't find them, at any rate.

Reply:
These should have been handled by `initObjects`

Reply:
I see. I'll remove them, then.